tests/e2e/cmp_test.go

```diff
@@ -0,0 +1,370 @@
+// +build !race
```

Comment:
Should we also move the following:

https://github.com/redhat-developer/odo/blob/c4cab72af8214fe7b9abc4fed1fa12648c60c9b3/tests/e2e/e2e_test.go#L65-L82

https://github.com/redhat-developer/odo/blob/c4cab72af8214fe7b9abc4fed1fa12648c60c9b3/tests/e2e/e2e_test.go#L120-L189

also here?

Reply:
@anmolbabu nope. We need a few basic component tests in e2e_test.go too.